### PR TITLE
Finished Events Page

### PR DESCRIPTION
**Pull request by hydraadra112**
- Created events page
- Created react calendar component for events page
- Created working carousel for events page
- Finished events page

### CODE REVIEW

File: frontend/package.json
Comment (hydraadra112) [inline frontend/package.json:1]: I installed react calendar

File: frontend/src/components/CalendarComponent.tsx
Comment (hydraadra112) [inline frontend/src/components/CalendarComponent.tsx:1]: A react calendar component. There is a link to the source code

File: frontend/src/components/events/Events_Carousel.tsx
Comment (hydraadra112) [inline frontend/src/components/events/Events_Carousel.tsx:1]: A carousel for the events page. For the ORBIT Recap

File: frontend/src/components/events/Events_EventDetails.tsx
Comment (hydraadra112) [inline frontend/src/components/events/Events_EventDetails.tsx:1]: A component for showcasing an event in the chosen date of user.

File: frontend/src/components/events/Events_RecapCard.tsx
Comment (hydraadra112) [inline frontend/src/components/events/Events_RecapCard.tsx:1]: Recap cards for the carousel

File: frontend/src/pages/ContactsUs.tsx
Comment (hydraadra112) [inline frontend/src/pages/ContactsUs.tsx:1]: Modified contact form to be dynamic in sizes

File: frontend/src/pages/Events.tsx
Comment (hydraadra112) [inline frontend/src/pages/Events.tsx:1]: The events page

**Comment by computerscience-person** [comment]
I'll comment on future changes nalang next time, but for now, I'll just merge this.

**Pull request merged**